### PR TITLE
Do not request apt-get installs bc and unzip

### PR DESCRIPTION
**Pull request by jayvdb**
They are part of the standard Travis precise and trusty images.

### CODE REVIEW

**Comment by jayvdb** [comment]
A build with this patch rebased onto the trusty patch: https://travis-ci.org/jayvdb/travis-oracle/builds/119504047

File: install.sh [hunk 1]
```diff
@@ -4,11 +4,13 @@
 [ -n "$ORACLE_HOME" ] || { echo "Missing ORACLE_HOME environment variable!"; exit 1; }
 
 ORACLE_RPM="$(basename $ORACLE_FILE .zip)"
+# Note that Travis ubuntu and trusty images contain bc and unzip
```
Comment (jayvdb) [inline install.sh]: gah, this should be: ... Travis Ubuntu precise and trusty images ...

**Comment by cbandy** [comment]
Fixed in 19c3924fc54584df83ecf7ad2d7e83d92186f3d3.

**Comment by jayvdb** [comment]
Umm, this wasnt done in 19c3924.  bc & unzip are still unnecessarily being requested to be installed.  Not a huge problem, if you dont want this change.

**Comment by cbandy** [comment]
> Umm, this wasnt done in 19c3924.

True. "Fixed" isn't what happened.

We need to `unzip` the RPM and the RPM postinstall depends on `bc`. I think its going to be easier to maintain if we ensure these utilities exist regardless of changes in Travis infrastructure.

The guard added in that commit ensures we only invoke `apt-get` when some dependency is missing.

**Pull request closed without merging**